profiles/makendrang.json

```diff
@@ -0,0 +1,8 @@
+![image](https://user-images.githubusercontent.com/74320000/185746011-a5c06825-183d-4dc3-abd5-bd057098381e.png)
```

Comment:
Please remove this line.
```suggestion
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient